drivers/syslog/Kconfig

```diff
@@ -195,7 +190,7 @@ config SYSLOG_CHAR
 
 config RAMLOG_SYSLOG
 	bool "Use RAMLOG for SYSLOG"
-	depends on RAMLOG && !ARCH_SYSLOG
+	depends on RAMLOG
```

Comment:
Missing default value for bool config

Reply:
the default is false, so we can skip it safely.

Reply:
I believe it is better to be explicit with Kconfig default values.
This ways we ensure the consistency of our Kconfig use across the codebase. Furthermore, it is easier to understand when you see the default field.

Reply:
Ok.

Reply:
Done!